packages/smooth_app/lib/pages/user_management/sign_up_page.dart

```diff
@@ -119,7 +119,8 @@ class _SignUpPageState extends State<SignUpPage> {
                 }
                 if (!UserManagementHelper.isUsernameLengthValid(
                     _userController.trimmedText)) {
-                  return 'Username must be less than 20 characters'; // TODO(vik4114): localization
+                  const int maxLength = OpenFoodAPIClient.USER_NAME_MAX_LENGTH;
+                  return 'Username must be less than $maxLength characters'; // TODO(vik4114): localization and translation
```

Comment:
This sentence must be translatable please

Reply:
I don't understand. Should I add this on en.arb file?

Reply:
yes you should go for it

Reply:
Okay

Reply:
@g123k Done.